screenpy/speech_tools.py

```diff
@@ -28,6 +28,8 @@ class name by replacing each capital letter with a space and a lower-case
     """
     if isinstance(describable, Describable):
         description = describable.describe()  # type: ignore # see PEP 544
+        if not description:
+            description = "something indescribable"
         description = description[0].lower() + description[1:]
         description = re.sub(r"[.,?!;:]*$", r"", description)
```

Comment:
We could probably put these two lines under an `else` and swap the condition, but the only advantage would be readability i think.

```python
        if description:
            description = description[0].lower() + description[1:]
            description = re.sub(r"[.,?!;:]*$", r"", description)
        else:
            description = "something indescribable"
```

Reply:
Good call. No reason to run through the regex in this case.